views/courses/components/tools.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="row">
 	<div class="col-sm-12">
-		<div class="list-group" data-courseId="{{@root/_id}}">
+		<div class="list-group" data-courseId="{{@root/_id}}" data-testid="course_tool_list">
```

Comment:
I actually meant to not use the same testid in "tools.hbs" and "add-tool.hbs" so they can be differentiated

Reply:
As I understamd it: data-testids help to address specific elements on a page in order to interact with them when executing end to end tests. They help to avoid the use of complex and fragile queries. They only have to be unique on a single page. But changed it according to your review.